kingsleyreadlexicon.tsv

```diff
@@ -4250,11 +4250,11 @@ ashford	𐑨𐑖𐑓𐑼𐑛	NP0	ˈæʃ.fəRd	115
 ashington	𐑨𐑖𐑦𐑙𐑑𐑩𐑯	NP0	ˈæʃ.ɪŋ.tən	70
 ashore	𐑩𐑖𐑹	AV0	əˈʃɔːR	647
 ashraf	𐑨𐑖𐑮𐑨𐑓	NP0	ˈæʃræf	23
-ashton-in-makerfield	𐑨𐑖𐑑𐑩𐑯-𐑦𐑯-𐑥𐑱𐑒𐑼𐑓𐑰𐑤𐑛	NP0	ˌæʃ.tən ɪn ˈmeɪ.kəR.fiːld	1
-ashton-under-lyne	𐑨𐑖𐑑𐑩𐑯-𐑳𐑯𐑛𐑼-𐑤𐑲𐑯	NP0	ˌæʃ.tən ˌʌn.dəR ˈlaɪn	30
+ashton-in-makerfield	𐑨𐑖𐑑𐑩𐑯-𐑦𐑯-𐑥𐑱𐑒𐑼𐑓𐑰𐑤𐑛	NP0	ˌæʃ.tən-ɪn-ˈmeɪ.kəR.fiːld	1
+ashton-under-lyne	𐑨𐑖𐑑𐑩𐑯-𐑳𐑯𐑛𐑼-𐑤𐑲𐑯	NP0	ˌæʃ.tən-ˌʌn.dəR-ˈlaɪn	30
 ashtray	𐑨𐑖𐑑𐑮𐑱	NN1	æʃtreɪ	210
 ashtrays	𐑨𐑖𐑑𐑮𐑱𐑟	NN2	æʃtreɪz	68
-ashworth	𐑨𐑖𐑢𐑻𐑔	NP0	ˈæʃ.wəRθ	148
+ashworth	𐑨𐑖𐑢𐑻𐑔	NP0	ˈæʃ.wɜːRθ	148
```

Comment:
To follow the first listed pronunciation in the Cambridge Pronouncing Dictionary I suggest:

`ashworth	𐑨𐑖𐑢𐑼𐑔	NP0	ˈæʃ.wəːRθ	148`

Reply:
I think this should be ‹ə› then, not ‹əː›.